data_management_system.rb

```diff
@@ -0,0 +1,87 @@
+# require 'pry'
+
+class Writer
+  def booking(book, _path, file)
+    data = {
+      title: book.title,
+      author: book.author
+    }
+    file.puts(data.to_json)
+    file.close
+  end
+
+  def book_writer(books, path)
+    file = if File.exist?(path)
+             File.open(path, 'a')
+           else
+             File.open(path, 'w')
+           end
+    books.each do |book|
+      booking(book, path, file) if book.is_a?(Book)
+    end
+  end
+
```

Comment:
- [ ] Same error with books, but my new book didn't get saved at all on the json 
![image](https://github.com/Lornakaboro/School-library/assets/62705947/b66cd3c5-454b-4874-9064-20619edbe9c1)